2024-01-04 01:57:28,502 T0000: NOTICE WORKER = gw2
2024-01-04 01:57:28,502 T0000: NOTICE using testbed file /data/work/logs/docker-master-446935-l2/testbed_1_2_1.yaml
2024-01-04 01:57:28,502 T0000: NOTICE Execution Start Time: 2024-01-04 01:57:28.500506
2024-01-04 01:57:28,502 T0000: NOTICE LOGS PATH: /data/work/logs/docker-master-446935-l2/gw2
2024-01-04 01:57:29,112 T0000: NOTICE VERSION: UNKNOWN UNKNOWN UNKNOWN
2024-01-04 01:57:29,112 T0000: NOTICE HOST: sonicmgmtmaster MEM: 376gb 48.8%
2024-01-04 01:57:29,112 T0000: NOTICE ROOT_PATH: /repo
2024-01-04 01:57:29,112 T0000: NOTICE Python: 3.8.10
2024-01-04 01:57:29,112 T0000: NOTICE Topology unknown: D1T1:4,D1D2:6,D2T1:4
2024-01-04 01:57:29,112 T0000: DEBUG 
SUITE ARGS: --load-image none --max-time module 9000 --max-time function 5400 --max-time session 3600 --logs-level=debug --topology-check=function --env SPYTEST_TESTBED_IGNORE_CONSTRAINTS TG,TGCARD,TGSPEED,CHIP,MODEL --noop --env SPYTEST_RESET_CONSOLES 0 --noop --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 1 --noop --env SPYTEST_RECOVERY_CR_PER_CMD_POST_OP 1 --noop --env SPYTEST_BATCH_POLL_STATUS_TIME 300 --noop --env SPYTEST_BATCH_DEAD_NODE_MAX_TIME 7200 --noop --breakout-mode none --env SPYTEST_SHUTDOWN_FREE_PORTS 1 --env SPYTEST_HOOKS_BREAKOUT_UITYPE click --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 0 --env SPYTEST_VERIFY_CSV_MIN_TOPOLOGY 1 --env SPYTEST_CMD_FAIL_RESULT_SUPPORT 0 --env SPYTEST_ROUTING_CONFIG_MODE split --env SPYTEST_CLEAR_DEVICE_METADATA_BGP_ASN 1 --env SPYTEST_TEXTFSM_USE_BUNDLED_PARSER 1 --feature-group master --ifname-type native --feature-disable bcmcmd --feature-disable warm-reboot --save config-db module --tclist-csv test_ft_add_unknownvlan_interface,test_ft_lldp_ConfigManAddrEntry,test_ft_lldp_LocManAddrEntry,test_ft_lldp_LocManAddrLen,test_ft_lldp_LocManAddrlfld,test_ft_lldp_docker_restart,test_ft_lldp_lldplocportid,test_ft_lldp_lldplocsysdesc,test_ft_lldp_lldplocsysname,test_ft_lldp_non_default_config,test_ft_lldp_rem_man_addr_table,test_ft_lldp_with_remote_server,test_ft_net_snmp_agent_mib,test_ft_net_snmp_vacm_mib,test_ft_ntp_disable_enable_with_message_log,test_ft_port_fn_verify_shut_noshut,test_ft_portchannel_with_vlan_variations,test_ft_snmp_coldstart_trap,test_ft_snmp_docker_restart,test_ft_snmp_dot1q_dot1db_mib,test_ft_snmp_dot1q_tp_fdb_address,test_ft_snmp_dot1q_tp_fdb_port,test_ft_snmp_dot3_stats_table,test_ft_snmp_ent_physical_table,test_ft_snmp_entity_mib_all,test_ft_snmp_framework_mib,test_ft_snmp_host_resource_mib,test_ft_snmp_icmp_Msgs,test_ft_snmp_if_mib_all,test_ft_snmp_ifx_table,test_ft_snmp_ipAddressRowStatus_ipv6,test_ft_snmp_ipAddressStorageType_ipv6,test_ft_snmp_ip_Address_Table,test_ft_snmp_ip_IfStats_Table,test_ft_snmp_ip_NetToPhysical_Table,test_ft_snmp_ip_System_Stats_Table,test_ft_snmp_ip_forward_table,test_ft_snmp_ipv6_If_Forward_default_HopLimit,test_ft_snmp_ipv6scope_index_table,test_ft_snmp_mib_2,test_ft_snmp_mpd_mib,test_ft_snmp_notification_mib,test_ft_snmp_root_node_walk,test_ft_snmp_snmpv2_mib,test_ft_snmp_sysDescr,test_ft_snmp_sysName,test_ft_snmp_target_mib,test_ft_snmp_tcp_mib,test_ft_snmp_test_syUpTime,test_ft_snmp_ucd_diskio_mib,test_ft_snmp_ucd_la_table,test_ft_snmp_ucd_system_stats,test_ft_snmp_udp_mib,test_ft_snmp_user_based_sm_mib,test_ft_snmp_view_based_acm_mib,test_ft_ssh_add_user_verify,test_ft_ssh_service_disable,test_ft_tacacs_ssh_login_with_tacacs_operations,test_ft_verify_logs_at_the_time_of_ssh_login_success,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,test_ft_vlan_delete_with_member,test_ft_vlan_save_config_warm_and_fast_reboot,test_ft_vlan_trunk_tagged,test_ft_watermark_telemetry_interval,test_lldp_verify_counters,test_ntp_exists_config,test_spytest_infra_first,test_spytest_infra_second,test_spytest_infra_last --noop

2024-01-04 01:57:29,112 T0000: NOTICE ARGS: --logs-level debug --results-prefix=results --get-tech-support none --fetch-core-files none --syslog-check none --save config-db module --topology-check module --logs-path logs/2024-01-04-01-57-17 --continue-on-collection-errors --max-time session 0 --max-time module 0 --max-time function 0 --noop --tryssh=0 --testbed-file testbed.yaml --env SPYTEST_RPS_DEBUG 0 --tclist-bucket 1,2,4 --test-suite community-vs-l2 --email rama.kristipati@broadcom.com --env SPYTEST_SCAPY_DBG_LVL 2 --noop --env SPYTEST_EMAIL_FROM SPyTest-VSNet@broadcom.com --noop --logs-path logs/docker-master-446935-l2 --env SPYTEST_EMAIL_BODY_PREFIX http://10.193.81.56:8080/sonic_mgmt_master/work/logs/docker-master-446935-l2/dashboard.html --email-subject-nargs VSNet-sonic_mgmt_master-DOCKER-L2-{{bldver}} : {{passrate}}: {{pass_cnt}}/{{executed}} docker-master-446935-l2 --noop --feature-group master
2024-01-04 01:57:29,771 T0000: DEBUG 'errors' not set in properties for V1 assuming 'default'
2024-01-04 01:57:29,771 T0000: DEBUG 'errors' not set in properties for V2 assuming 'default'
2024-01-04 01:57:29,771 T0000: INFO  TGEN-IP 127.0.0.1
2024-01-04 01:57:29,772 T0000: WARN  Registering Topology
2024-01-04 01:57:29,802 T0000: INFO  Connecting all devices
2024-01-04 01:57:29,803 T0000: INFO  Loading scapy:1.0 127.0.0.1 Ports: ['1/1', '1/2', '1/3', '1/4', '1/5', '1/6', '1/7', '1/8'] skip: False
2024-01-04 01:57:30,400 T0006: INFO  [D2-V2] Connecting to device (V2): sonic_ssh: 172.18.0.3:22 ..
2024-01-04 01:57:30,401 T0006: INFO  [D2-V2] TRY Authentication: 172.18.0.3:22 admin YourPaSsWoRd
2024-01-04 01:57:30,407 T0005: INFO  [D1-V1] Connecting to device (V1): sonic_ssh: 172.18.0.2:22 ..
2024-01-04 01:57:30,408 T0005: INFO  [D1-V1] TRY Authentication: 172.18.0.2:22 admin YourPaSsWoRd
2024-01-04 01:57:30,833 T0000: INFO  TG Scapy Init
2024-01-04 01:57:30,833 T0000: INFO  TG Base Init skip: False
2024-01-04 01:57:30,839 T0000: INFO  TG Scapy Connect 127.0.0.1:8009
2024-01-04 01:57:30,929 T0000: INFO  Server Logs Path /data/work/scapy-tgen/logs/current/inst-2024_01_04_01_57_30_863146/gw2/default.log
2024-01-04 01:57:30,936 T0000: DEBUG REQ: tg_connect(port_list=['1/1', '1/2', '1/3', '1/4', '1/5', '1/6', '1/7', '1/8'])
2024-01-04 01:57:35,617 T0005: DEBUG [D1-V1] Connected ...
2024-01-04 01:57:35,621 T0006: DEBUG [D2-V2] Connected ...
2024-01-04 01:57:36,446 T0006: INFO  [D2-V2] Prompt at the connection start: 'admin@V2:~$' 
2024-01-04 01:57:36,448 T0006: AUDIT [D2-V2] ['export TMOUT=0', 'stty cols 5000', 'uptime']
2024-01-04 01:57:36,451 T0005: INFO  [D1-V1] Prompt at the connection start: 'admin@V1:~$' 
2024-01-04 01:57:36,452 T0005: AUDIT [D1-V1] ['export TMOUT=0', 'stty cols 5000', 'uptime']
2024-01-04 01:57:36,859 T0006: INFO  [D2-V2] FCMD: export TMOUT=0;stty cols 5000;uptime
2024-01-04 01:57:36,866 T0005: INFO  [D1-V1] FCMD: export TMOUT=0;stty cols 5000;uptime
2024-01-04 01:57:37,123 T0006: INFO  [D2-V2]  01:57:36 up 27 days,  3:02,  1 user,  load average: 64.72, 25.47, 12.68
2024-01-04 01:57:37,123 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:57:37,133 T0006: AUDIT [D2-V2] cat /proc/uptime
2024-01-04 01:57:37,136 T0005: INFO  [D1-V1]  01:57:36 up 27 days,  3:02,  1 user,  load average: 64.72, 25.47, 12.68
2024-01-04 01:57:37,137 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:57:37,139 T0005: AUDIT [D1-V1] cat /proc/uptime
2024-01-04 01:57:37,539 T0006: INFO  [D2-V2] FCMD: cat /proc/uptime
2024-01-04 01:57:37,544 T0005: INFO  [D1-V1] FCMD: cat /proc/uptime
2024-01-04 01:57:37,796 T0006: INFO  [D2-V2] 2343744.26 184751508.59
2024-01-04 01:57:37,796 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:57:37,798 T0006: INFO  [D2-V2] UPTIME: 2343744.26
2024-01-04 01:57:37,799 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:37,802 T0005: INFO  [D1-V1] 2343744.27 184751508.96
2024-01-04 01:57:37,802 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:57:37,803 T0005: INFO  [D1-V1] UPTIME: 2343744.27
2024-01-04 01:57:37,804 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:38,206 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:38,210 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:45,661 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:57:45,662 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:57:45,663 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:57:45,663 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:57:45,666 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:45,668 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:45,669 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:45,669 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:57:45,670 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:45,670 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:57:50,672 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:50,673 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:51,088 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:51,093 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:51,851 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:57:51,851 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:57:51,854 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:51,856 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:57:51,857 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:57:51,859 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:51,859 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:57:51,861 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:51,863 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:51,863 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:57:56,864 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:56,869 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:57,270 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:57,277 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:58,181 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:57:58,181 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:57:58,184 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:58,186 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:58,186 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:57:58,189 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:57:58,189 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:57:58,191 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:57:58,193 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:57:58,193 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:00,241 T0000: DEBUG RESP: tg_connect {"port_handle": {"1/1": "port-1", "1/2": "port-2", "1/3": "port-3", "1/4": "port-4", "1/5": "port-5", "1/6": "port-6", "1/7": "port-7", "1/8": "port-8"}, "status": 1}
2024-01-04 01:58:00,258 T0000: INFO  TG Connection: Success
2024-01-04 01:58:00,258 T0000: INFO  TG Connect...done
2024-01-04 01:58:03,189 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:03,196 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:03,597 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:03,603 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:04,257 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:04,257 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:04,259 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:04,260 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:04,261 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:04,363 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:04,364 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:04,366 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:04,368 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:04,368 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:09,264 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:09,374 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:09,672 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:09,782 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:10,434 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:10,434 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:10,436 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:10,438 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:10,438 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:10,492 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:10,492 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:10,495 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:10,497 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:10,497 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:15,440 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:15,500 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:15,848 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:15,907 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:16,609 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:16,610 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:16,613 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:16,614 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:16,614 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:16,668 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:16,669 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:16,671 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:16,673 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:16,673 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:21,616 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:21,676 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:22,025 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:22,083 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:22,736 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:22,737 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:22,739 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:22,741 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:22,741 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:22,795 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:22,795 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:22,799 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:22,802 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:22,802 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:27,747 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:27,804 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:28,155 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:28,212 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:28,916 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:28,917 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:28,919 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:28,921 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:28,922 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:28,973 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:28,974 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:28,976 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:28,978 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:28,978 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:33,925 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:33,980 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:34,334 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:34,388 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:35,146 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:35,146 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:35,149 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:35,151 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:35,151 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:35,153 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:35,154 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:35,156 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:35,158 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:35,158 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:40,156 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:40,160 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:40,564 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:40,570 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:41,381 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:41,382 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:41,386 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:41,386 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:41,389 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:41,391 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:41,393 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:41,393 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:41,394 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:41,395 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:46,399 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:46,400 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:46,814 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:46,819 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:47,478 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:47,479 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:47,481 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:47,482 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:47,484 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:47,486 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:47,488 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:47,488 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:47,489 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:47,489 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:52,492 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:52,493 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:52,907 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:52,912 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:53,620 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:53,620 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:53,623 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:53,624 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:53,626 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:53,629 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:53,631 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:53,631 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:53,634 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:53,635 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:58,637 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:58,640 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:59,047 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:59,052 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:59,713 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:58:59,713 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:58:59,716 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:58:59,716 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:58:59,718 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:59,720 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:58:59,721 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:59,721 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:58:59,722 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:58:59,722 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:04,724 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:04,725 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:05,139 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:05,145 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:05,855 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:05,856 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:05,858 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:05,859 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:05,861 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:05,864 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:05,866 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:05,866 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:05,867 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:05,867 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:10,872 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:10,873 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:11,281 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:11,284 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:11,940 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:11,940 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:11,942 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:11,944 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:11,944 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:11,945 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:11,945 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:11,947 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:11,948 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:11,948 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:16,951 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:16,953 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:17,365 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:17,371 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:18,081 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:18,081 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:18,084 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:18,084 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:18,087 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:18,089 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:18,091 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:18,091 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:18,092 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:18,092 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:23,096 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:23,098 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:23,512 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:23,518 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:24,276 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:24,276 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:24,279 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:24,281 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:24,281 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:24,285 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:24,285 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:24,287 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:24,289 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:24,289 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:29,288 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:29,292 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:29,696 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:29,702 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:30,414 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:30,415 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:30,417 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:30,418 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:30,420 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:30,422 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:30,424 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:30,424 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:30,425 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:30,425 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:35,428 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:35,431 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:35,838 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:35,842 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:36,499 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:36,500 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:36,503 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:36,503 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:36,506 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:36,508 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:36,510 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:36,511 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:36,512 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:36,512 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:41,512 T0006: AUDIT [D2-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:41,516 T0005: AUDIT [D1-V1] sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:41,920 T0006: INFO  [D2-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:41,927 T0005: INFO  [D1-V1] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:42,585 T0006: INFO  [D2-V2] Error response from daemon: No such container: bgp
2024-01-04 01:59:42,585 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:42,587 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:42,589 T0005: INFO  [D1-V1] Error response from daemon: No such container: bgp
2024-01-04 01:59:42,589 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:42,592 T0006: WARN  [D2-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:42,592 T0006: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:42,594 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-04 01:59:42,596 T0005: WARN  [D1-V1] Failed to read hostname from vtysh retry again in 5 sec
2024-01-04 01:59:42,596 T0005: INFO  Sleep for 5 sec(s)... @371
2024-01-04 01:59:47,598 T0006: ERROR [D2-V2] Failed to read hostname from vtysh - assuming sonic
2024-01-04 01:59:47,599 T0006: INFO  [D2-V2] Base Prompt: sonic
2024-01-04 01:59:47,600 T0006: DEBUG [D2-V2] Wait for system ready Ref: 866
2024-01-04 01:59:47,604 T0005: ERROR [D1-V1] Failed to read hostname from vtysh - assuming sonic
2024-01-04 01:59:47,606 T0005: INFO  [D1-V1] Base Prompt: sonic
2024-01-04 01:59:47,606 T0005: DEBUG [D1-V1] Wait for system ready Ref: 866
2024-01-04 01:59:48,007 T0006: INFO  [D2-V2] FCMD: sudo md5sum /etc/spytest/remote/spytest-helper.py
2024-01-04 01:59:48,012 T0005: INFO  [D1-V1] FCMD: sudo md5sum /etc/spytest/remote/spytest-helper.py
2024-01-04 01:59:48,366 T0006: INFO  [D2-V2] md5sum: /etc/spytest/remote/spytest-helper.py: No such file or directory
2024-01-04 01:59:48,366 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:48,370 T0006: INFO  [D2-V2] MD5 different SRC: 4522abaafb8e29890a8377a810228525 DST: []
2024-01-04 01:59:48,371 T0005: INFO  [D1-V1] md5sum: /etc/spytest/remote/spytest-helper.py: No such file or directory
2024-01-04 01:59:48,372 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:48,375 T0006: INFO  [D2-V2] Trying to upload file '/repo/spytest/remote/spytest-helper.py'. attempt '1'
2024-01-04 01:59:48,376 T0006: INFO  [D2-V2] Transfer: SRC(SVR/VDI): /repo/spytest/remote/spytest-helper.py DST(V2): /tmp/spytest-helper.py
2024-01-04 01:59:48,378 T0005: INFO  [D1-V1] MD5 different SRC: 4522abaafb8e29890a8377a810228525 DST: []
2024-01-04 01:59:48,379 T0006: ERROR [D2-V2] No mgmt IP
2024-01-04 01:59:48,380 T0005: INFO  [D1-V1] Trying to upload file '/repo/spytest/remote/spytest-helper.py'. attempt '1'
2024-01-04 01:59:48,381 T0006: INFO  [D2-V2] Doing SFTP transfer IP = None/YourPaSsWoRd/broadcom
2024-01-04 01:59:48,393 T0005: INFO  [D1-V1] Transfer: SRC(SVR/VDI): /repo/spytest/remote/spytest-helper.py DST(V1): /tmp/spytest-helper.py
2024-01-04 01:59:48,394 T0005: ERROR [D1-V1] No mgmt IP
2024-01-04 01:59:48,395 T0005: INFO  [D1-V1] Doing SFTP transfer IP = None/YourPaSsWoRd/broadcom
2024-01-04 01:59:48,848 T0006: INFO  [D2-V2] FCMD: sudo md5sum /tmp/spytest-helper.py
2024-01-04 01:59:48,854 T0005: INFO  [D1-V1] FCMD: sudo md5sum /tmp/spytest-helper.py
2024-01-04 01:59:49,156 T0006: INFO  [D2-V2] 4522abaafb8e29890a8377a810228525  /tmp/spytest-helper.py
2024-01-04 01:59:49,157 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:49,159 T0005: INFO  [D1-V1] 4522abaafb8e29890a8377a810228525  /tmp/spytest-helper.py
2024-01-04 01:59:49,160 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:49,161 T0006: INFO  [D2-V2] Trying to copy file to '/etc/spytest/remote/spytest-helper.py'. attempt '1'
2024-01-04 01:59:49,162 T0005: INFO  [D1-V1] Trying to copy file to '/etc/spytest/remote/spytest-helper.py'. attempt '1'
2024-01-04 01:59:49,163 T0006: INFO  [D2-V2] SCMD: sudo mkdir -p /etc/spytest/remote && sudo cp -f /tmp/spytest-helper.py /etc/spytest/remote/spytest-helper.py
2024-01-04 01:59:49,167 T0005: INFO  [D1-V1] SCMD: sudo mkdir -p /etc/spytest/remote && sudo cp -f /tmp/spytest-helper.py /etc/spytest/remote/spytest-helper.py
2024-01-04 01:59:51,773 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:51,776 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:51,778 T0006: WARN  [D2-V2] Execute 'sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300' with timeout 600 sec -- TRY 1
2024-01-04 01:59:51,780 T0005: WARN  [D1-V1] Execute 'sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300' with timeout 600 sec -- TRY 1
2024-01-04 01:59:51,782 T0006: INFO  [D2-V2] SCMD: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:51,785 T0005: INFO  [D1-V1] SCMD: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:55,616 T0005: WARN  [D1-V1] Skipped error checking in session init but detected pattern: containing_python_traceback command: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:55,618 T0006: WARN  [D2-V2] Skipped error checking in session init but detected pattern: containing_python_traceback command: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1] 
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1] Remote CMD: 'show platform summary'
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1] Error: Failed to execute 'show platform summary' (out: '') (err: 'Traceback (most recent call last):
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]   File "/usr/local/bin/show", line 5, in <module>
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]     from show.main import cli
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]     if is_gearbox_configured():
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,619 T0005: INFO  [D1-V1]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1]     app_db.connect(app_db.APPL_DB)
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1] 
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1] Remote CMD: 'cat /usr/share/sonic/device/None/None/port_config.ini | tail -1'
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1] Remote CMD: 'sudo show interfaces status '
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1] Error: Failed to execute 'sudo show interfaces status ' (out: '') (err: 'Traceback (most recent call last):
2024-01-04 01:59:55,620 T0005: INFO  [D1-V1]   File "/usr/local/bin/show", line 5, in <module>
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]     from show.main import cli
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]     if is_gearbox_configured():
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]     app_db.connect(app_db.APPL_DB)
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,621 T0005: INFO  [D1-V1] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-04 01:59:55,622 T0005: INFO  [D1-V1] 
2024-01-04 01:59:55,622 T0005: INFO  [D1-V1] admin@V1:~$ 
2024-01-04 01:59:55,622 T0006: INFO  [D2-V2] 
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2] Remote CMD: 'show platform summary'
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2] Error: Failed to execute 'show platform summary' (out: '') (err: 'Traceback (most recent call last):
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]   File "/usr/local/bin/show", line 5, in <module>
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]     from show.main import cli
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]     if is_gearbox_configured():
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-04 01:59:55,623 T0006: INFO  [D2-V2]     app_db.connect(app_db.APPL_DB)
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2] 
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2] Remote CMD: 'cat /usr/share/sonic/device/None/None/port_config.ini | tail -1'
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2] Remote CMD: 'sudo show interfaces status '
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2] Error: Failed to execute 'sudo show interfaces status ' (out: '') (err: 'Traceback (most recent call last):
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2]   File "/usr/local/bin/show", line 5, in <module>
2024-01-04 01:59:55,624 T0006: INFO  [D2-V2]     from show.main import cli
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]     if is_gearbox_configured():
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]     app_db.connect(app_db.APPL_DB)
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-04 01:59:55,625 T0006: INFO  [D2-V2]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-04 01:59:55,626 T0006: INFO  [D2-V2] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-04 01:59:55,626 T0006: INFO  [D2-V2] 
2024-01-04 01:59:55,626 T0006: INFO  [D2-V2] admin@V2:~$ 
2024-01-04 01:59:55,627 T0005: ERROR [D1-V1] Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:55,628 T0006: ERROR [D2-V2] Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-04 01:59:55,628 T0004: ERROR exception in thread: Traceback (most recent call last):
  File "/repo/utilities/parallel.py", line 159, in _thread_func
    retvals[index] = func(*args, **kwargs)
  File "/repo/spytest/net.py", line 1064, in connect_to_device
    show_ver_output = self.connect_to_device_real(access, retry, recon, recover)
  File "/repo/spytest/net.py", line 1290, in connect_to_device_real
    self.do_common_init(devname, phase=0, max_ready_wait=max_ready_wait)
  File "/repo/spytest/net.py", line 866, in do_common_init
    sys_status = self.wa.wait_system_status(devname, max_time=max_ready_wait)
  File "/repo/spytest/framework.py", line 1616, in wait_system_status
    return self._wait_for_ports(dut, max_time)
  File "/repo/spytest/framework.py", line 1605, in _wait_for_ports
    return self.net.apply_remote(dut, "wait-for-ports", [max_time])
  File "/repo/spytest/net.py", line 4523, in apply_remote
    raise ValueError(msg)
ValueError: Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  

2024-01-04 01:59:55,628 T0004: ERROR exception in thread: Traceback (most recent call last):
  File "/repo/utilities/parallel.py", line 159, in _thread_func
    retvals[index] = func(*args, **kwargs)
  File "/repo/spytest/net.py", line 1064, in connect_to_device
    show_ver_output = self.connect_to_device_real(access, retry, recon, recover)
  File "/repo/spytest/net.py", line 1290, in connect_to_device_real
    self.do_common_init(devname, phase=0, max_ready_wait=max_ready_wait)
  File "/repo/spytest/net.py", line 866, in do_common_init
    sys_status = self.wa.wait_system_status(devname, max_time=max_ready_wait)
  File "/repo/spytest/framework.py", line 1616, in wait_system_status
    return self._wait_for_ports(dut, max_time)
  File "/repo/spytest/framework.py", line 1605, in _wait_for_ports
    return self.net.apply_remote(dut, "wait-for-ports", [max_time])
  File "/repo/spytest/net.py", line 4523, in apply_remote
    raise ValueError(msg)
ValueError: Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  

2024-01-04 01:59:55,630 T0004: ERROR [D1-V1] Failed to connect to device
2024-01-04 01:59:55,631 T0004: ERROR [D2-V2] Failed to connect to device
2024-01-04 01:59:55,631 T0000: ERROR Failed to connect (['V1', 'V2'])
2024-01-04 01:59:55,632 T0000: INFO  --FINISH--RUN--4 @559
